README.md

```diff
@@ -18,6 +18,7 @@ Prequisites:
 * Postgres installed and running -- on MacOS, I like https://postgresapp.com/
 * `yarn` installed for webpacker -- on MacOS, `brew install yarn`.
 * vips installed --  on MacOS `brew install vips`
+* Have your AWS credentials put in the default profile at ~/.aws/config
 
```

Comment:
i guess running a cap deploy didn't AWS credentials at all before?

This is a standard place to put AWS credentials?

For a developer running an app locally, we keep AWS credentials somewhere else (local_env.yml or your bash ENV). Not sure if having two places is gonna be a problem... especially because the AWS credentials we use to run a dev app may _not_ have permission to do what's needed here! May need more thought, my head is swimming. 

But I already have a `AWS_SECRET_ACCESS_KEY` and `AWS_ACCESS_KEY_ID` -- how does that interact with `~/.aws/config`?